doc/rbd/rbd-mirroring.rst

```diff
@@ -23,16 +23,19 @@ capability is available in two modes:
   blocks can be quickly determined without the need to scan the full RBD image.
   Since this mode is not as fine-grained as journaling, the complete delta 
   between two snapshots will need to be synced prior to use during a failover
-  scenario. Any partially applied set of deltas will be rolled back at moment
-  of failover.
+  scenario. Any partially applied set of deltas will be rolled back at the 
+  moment of failover.
 
 .. note:: journal-based mirroring requires the Ceph Jewel release or later;
    snapshot-based mirroring requires the Ceph Octopus release or later.
 
+.. note:: All instances of the term "namespace" in this document refer to RBD
+   namespaces.
+
 Mirroring is configured on a per-pool basis within peer clusters and can be
-configured on a specific subset of images within the pool.  You can also mirror
-all images within a given pool when using journal-based
-mirroring. Mirroring is configured using the ``rbd`` command. The
+configured on an RBD namespace or specific subset of images within the pool or
+namespace. You can also mirror all images within a given pool when using
```

Comment:
`rbd mirror pool enable pool1/ns1 pool` with journal-based mirroring would enable it for all images in the namespace, right?
```suggestion
namespace. You can also mirror all images within a given pool or namespace when using
```